tests/queries/0_stateless/02167_format_from_file_extension.sh

```diff
@@ -5,6 +5,12 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
+function cleanup()
+{
+    user_files_path=$(clickhouse-client --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep Exception | awk '{gsub("/nonexist.txt","",$9); print $9}')
```

Comment:
Also I'd suggest to add a comment that this command expects an error message like 
`Code: 107. DB::Exception: Received from localhost:9000. DB::Exception: File /home/davenger/src/ClickHouse/build_Debug/user_files/nonexist.txt doesn't exist. (FILE_DOESNT_EXIST)`
So that if this message changes in future it will be easier to fix this command
```suggestion
    user_files_path=$($CLICKHOUSE_CLIENT --query "select _path,_file from file('nonexist.txt', 'CSV', 'val1 char')" 2>&1 | grep FILE_DOESNT_EXIST | awk '{gsub("/nonexist.txt","",$9); print $9}')
```